CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_CI_OPTC.uc

```diff
@@ -0,0 +1,124 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: Modifies Templates in order to make infinite items single build instead
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+class X2Helper_CI_OPTC extends Object config (SingleBuildItems);
+
+struct SingleBuildItem
+{
+	var name SchematicName;
+	var int TradingPostValue;
+	
+	var array<StrategyCost> Costs;
+};
+
+var config array<SingleBuildItem> SingleBuildItems;
+
+static function ModifyTemplates()
+{
+	local array<X2DataTemplate> DifficultyVariants;
+	local X2ItemTemplateManager TemplateManager;
+	local X2SchematicTemplate Schematic;
+	local X2ItemTemplate Template;
+	local SingleBuildItem Item;
```

Comment:
Please rename to `ItemBuildInfo` otherwise the code below is very confusing to read